README.md

```diff
@@ -1,2 +1,46 @@
-# Portfolio-setup-and-mobile-first
-For the first milestone in the process of creating your portfolio website, you will:  set up a new repository and prepare it for development using best practices (e.g. linters). build the first 2 sections of the mobile website using the template you chose in the previous step.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio-setup-and-mobile-first] <a name="about-project"></a>
+
+For the first milestone in the process of creating your portfolio website, you will: set up a new repository and prepare it for development using best practices (e.g. linters). build the first 2 sections of the mobile website using the template you chose in the previous step.
+
+
+## 👥 Author <a name="authors"></a>
+
+
+👤 **Mladan**
+
+- GitHub: [@djo1975](https://github.com/djo1975)
+g
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly make sure your README is documented in a professional way. Please do not remove all the sections that are applicable to your project from the template.
`Set up` `Get started` `Tech stack` 
`Kinldy make sure you have those in your README`